client/X11/xf_gdi.c

```diff
@@ -1076,7 +1084,6 @@ static BOOL xf_gdi_surface_bits(rdpContext* context, const SURFACE_BITS_COMMAND*
 
 		default:
 			WLog_ERR(TAG, "Unsupported codecID %" PRIu16 "", cmd->bmp.codecID);
-			ret = TRUE;
 			goto fail;
```

Comment:
shall we fail and disconnect when we receive an unknown codec ?

Reply:
well, yes.
otherwise you'll end up with a very distorted screen anyway.

Reply:
hum, ok I'm seeing it more like "I've not announced that codec, so why are you using it with me ?". Ok.